assets/src/adaptivity/scripting.ts

```diff
@@ -1,19 +1,19 @@
 import { EntityId } from '@reduxjs/toolkit';
 import { Environment, Evaluator, Lexer, Parser } from 'janus-script';
-import flatten from 'lodash/flatten';
-import uniq from 'lodash/uniq';
+// import flatten from 'lodash/flatten';
+// import uniq from 'lodash/uniq';
 import { parseArray, parseBoolean } from 'utils/common';
 import { CapiVariableTypes, getCapiType } from './capi';
 import { janus_std } from './janus-scripts/builtin_functions';
 
-let conditionsNeedEvaluation: string[] = [];
+//const conditionsNeedEvaluation: string[] = [];
 export const setConditionsWithExpression = (facts: string[]) => {
-  conditionsNeedEvaluation.push(...facts);
-  conditionsNeedEvaluation = uniq(flatten(Array.from(new Set(conditionsNeedEvaluation))));
-  const script = `let session.conditionsNeedEvaluation = ${JSON.stringify(
-    conditionsNeedEvaluation,
-  )}`;
-  evalScript(script, defaultGlobalEnv);
+  // conditionsNeedEvaluation.push(...facts);
```

Comment:
This was done to store the conditions that Needed Evaluation. Commenting out this as well.